app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/dashboard/DashboardFragment.kt

```diff
@@ -38,12 +40,17 @@ class DashboardFragment : MifosBaseFragment(), View.OnClickListener {
         rootView = inflater.inflate(R.layout.fragment_dashboard, container, false)
         setHasOptionsMenu(true)
         setToolbarTitle(getString(R.string.home))
+        if (arguments != null) {
+            1
+            customerIdentification = arguments!!.getString(ConstantKeys.CUSTOMER_IDENTIFIER)
```

Comment:
I think it will be much better if you something like this:
`arguments?.let{
customerIdentification = it.getString(ConstantKeys.CUSTOMER_IDENTIFIER)
}`
Using `!!` here is redundant because you have already checked for its nullability on line number 43.